src/components/Documentation/schemaDocs.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import { useAppSelector } from "~/rtk";
+
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "~/components/ui";
+
+const SchemaDoc = () => {
+  const { schema } = useAppSelector((state) => state);
```

Comment:
wrong usage of useSelector
you need to make useSelector(state => state.schema 

to only subscribe on schema state property

Reply:
please look through all usages of useSelector in the app and fix